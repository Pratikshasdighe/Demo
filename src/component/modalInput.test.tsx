import React from "react";
import renderer from "react-test-renderer";
import { NativeBaseProvider } from "native-base";

import ModelInput from "./modalInput";

it("expect to render ModalInput", () => {
  const modalInputSnap = renderer
    .create(
      <NativeBaseProvider>
        <ModelInput />
      </NativeBaseProvider>
    )
    .toJSON();
  expect(modalInputSnap).toMatchSnapshot();
});
