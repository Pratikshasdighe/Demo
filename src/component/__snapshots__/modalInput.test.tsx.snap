// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render ModalInput 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
/>
`;
